src/Shouldly/Shouldly.csproj

```diff
@@ -21,7 +20,7 @@
     <Optimize>false</Optimize>
   </PropertyGroup>
   <PropertyGroup Condition="'$(TargetFramework)'== 'net451'">
-      <FrameworkPathOverride>$(NuGetPackageRoot)microsoft.targetingpack.netframework.v4.5.1/1.0.1/lib/net451/</FrameworkPathOverride>
+      <FrameworkPathOverride>$(NuGetPackageRoot)microsoft.netframework.referenceassemblies.net451/1.0.0-preview.1/build/.NETFramework/v4.5.1</FrameworkPathOverride>
```

Comment:
This property can go now, as it is disabled explicitly by the ReferenceAssemblies package

Reply:
I did wonder this, but it blows up if I remove it.

Reply:
Interesting, was that locally or on CI?

Reply:
Just tried it and it works here. I did a write-up on how it works:
https://stu.dev/dotnet-framework-reference-assemblies-via-nuget/
`FrameworkPathOverride` should do nothing when this package is referenced.

Reply:
> Interesting, was that locally or on CI?

Weird, tried it and it worked for some reason.